src/redux/categories/categories.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import NavBar from '../components/header/navigation';
+import NavBar from '../../components/header/navigation';
 
 const Categories = () => (
   <div className="container">
```

Comment:
- [ ] It would be nice if this file is moved to components, let's redux hold all of redux related logics, and anything related to components constructions and displays stay in components folder for the sake of clean codes and readability